### PR TITLE
Netatmo 1070

### PR DESCRIPTION
**Pull request by clinique**

### CODE REVIEW

**Comment by clinique** [comment]
@lolodomo  : for your review

**Comment by lolodomo** [comment]
Fixes issue #3316 
Fixes issue #3317 
Fixes issue #3584

**Comment by lolodomo** [comment]
Does this new retrofit lib version include stuff for presence camera ?

**Comment by clinique** [comment]
@lolodomo : no, this does not include presence. I do not own one currently, so not able to test anything.

**Comment by RickyTR** [comment]
@clinique I tried the latest pulled version and I saw that you introduced the possibility to change the refresh period for thermostat as it was in older versions. I tried to change the setting but the refresh is still happening every hour. Can you check it, please?

File: addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/channelhelper/BatteryHelper.java [hunk 1]
```diff
@@ -55,16 +50,15 @@ public void setModule(Object module) {
             try {
                 if (CHANNEL_BATTERY_LEVEL.equalsIgnoreCase(channelId)
```
[the remark this answers is not in the document]
Comment (clinique) [reply addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/channelhelper/BatteryHelper.java]: You're right ! Will change this on next occasion to work on this binding

**Pull request merged**